app/controllers/donors_controller.rb

```diff
@@ -2,6 +2,7 @@ class DonorsController < ApplicationController
   # GET donors/:id
   def show
     @donor = find_donor(params[:id])
+    @donation = find_donation(params[:id])
```

Comment:
Hm. We're in the `DonorsController` here, and `params[:id]` is a `Donor` id ... 😅